### PR TITLE
I change the file and give it to you 

### PR DESCRIPTION
**Pull request by Jfef**
take easy ,this is a easy file

### CODE REVIEW

File: gitHubLearn.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
 Practice new file modify
 xiugai
 修改新的一行，接受提交
+modify changes
```
Comment (neonmike) [inline gitHubLearn.txt]: good modify ,think this right

File: gitHubLearn.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+差异化修改
+pull request
```
Comment (neonmike) [inline gitHubLearn.txt]: remove this

**Comment by neonmike** [comment]
if you modify file according this ,I will merge

**Comment by Jfef** [comment]
I change the will which you said to me

**Comment by neonmike** [comment]
goog jobs

**Pull request merged**